{
    "dataType": "CVE_RECORD",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2025-48432",
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "dateUpdated": "2025-06-11T14:59:01.028Z",
        "dateReserved": "2025-05-21T00:00:00.000Z",
        "datePublished": "2025-06-05T00:00:00.000Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unknown",
                    "product": "Django",
                    "vendor": "djangoproject",
                    "versions": [
                        {
                            "lessThan": "4.2.23",
                            "status": "affected",
                            "version": "4.2",
                            "versionType": "custom"
                        },
                        {
                            "lessThan": "5.1.11",
                            "status": "affected",
                            "version": "5.1",
                            "versionType": "custom"
                        },
                        {
                            "lessThan": "5.2.3",
                            "status": "affected",
                            "version": "5.2",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "cpeApplicability": [
                {
                    "nodes": [
                        {
                            "operator": "OR",
                            "negate": false,
                            "cpeMatch": [
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "4.2",
                                    "versionEndExcluding": "4.2.23"
                                },
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "5.1",
                                    "versionEndExcluding": "5.1.11"
                                },
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "5.2",
                                    "versionEndExcluding": "5.2.3"
                                }
                            ]
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in Django 5.2 before 5.2.3, 5.1 before 5.1.11, and 4.2 before 4.2.23. Internal HTTP response logging does not escape request.path, which allows remote attackers to potentially manipulate log output via crafted URLs. This may lead to log injection or forgery when logs are viewed in terminals or processed by external systems."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-117",
                            "description": "CWE-117 Improper Output Neutralization for Logs",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre",
                "dateUpdated": "2025-06-11T14:59:01.028Z"
            },
            "references": [
                {
                    "url": "https://docs.djangoproject.com/en/dev/releases/security/"
                },
                {
                    "url": "https://groups.google.com/g/django-announce"
                },
                {
                    "url": "https://www.djangoproject.com/weblog/2025/jun/04/security-releases/"
                },
                {
                    "url": "https://www.djangoproject.com/weblog/2025/jun/10/bugfix-releases/"
                }
            ],
            "x_generator": {
                "engine": "enrichogram 0.0.1"
            },
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "baseScore": 4,
                        "baseSeverity": "MEDIUM",
                        "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:N/I:L/A:N"
                    }
                }
            ]
        },
        "adp": [
            {
                "title": "CVE Program Container",
                "references": [
                    {
                        "url": "http://www.openwall.com/lists/oss-security/2025/06/04/5"
                    },
                    {
                        "url": "http://www.openwall.com/lists/oss-security/2025/06/10/2"
                    },
                    {
                        "url": "http://www.openwall.com/lists/oss-security/2025/06/10/3"
                    },
                    {
                        "url": "http://www.openwall.com/lists/oss-security/2025/06/10/4"
                    }
                ],
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2025-06-10T18:03:40.110Z"
                }
            },
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-06-05T13:20:12.712693Z",
                                "id": "CVE-2025-48432",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-06-05T14:10:52.025Z"
                }
            }
        ]
    },
    "dataVersion": "5.1"
}